src/pointer-analysis/dereference_callback.h

```diff
@@ -25,7 +25,7 @@ class symbolt;
 class dereference_callbackt
 {
 public:
-  virtual ~dereference_callbackt();
+  virtual ~dereference_callbackt() = default;
```

Comment:
Maybe compilers and/or linkers have gotten better at this, but the point of this file was that it would provide a place to store the generated virtual table. Maybe today that's no longer necessary?